jupyterhub/tasks/main.yml.bak

```diff
@@ -0,0 +1,385 @@
+---
+# file: roles/jupyter/tasks/main.yml
+# requires python3
+
+####################################
+# Install base dependencies
```

Comment:
Why is this file here? Doesn't git already have a backup of the previous version of this file?

Reply:
Out of habit... when I make changes to a file, I create a .bak and in this case, I would have deleted this file if I was certain that the install was successful. Me being overly cautious.

Reply:
You do not need to keep a copy of the original file. Since we are using a git repository you can get back to what the original file looked like. You can even see all the different versions of a file. For example here all versions of [jupyterhub/tasks/main.yaml](https://github.com/Duke-GCB/gcb-ansible-roles/commits/master/jupyterhub/tasks/main.yml) from this repo.